sklearn/linear_model/coordinate_descent.py

```diff
@@ -389,11 +392,10 @@ def enet_path(X, y, l1_ratio=0.5, eps=1e-3, n_alphas=100, alphas=None,
 
     # X should be normalized and fit already if function is called
     # from ElasticNet.fit
-    if pre_fit:
```

Comment:
The `pre_fit` flag was used so that `enet_path` does not run `_pre_fit` a second time when called from `ElasticNet.fit`.